fsspec/dircache.py

```diff
@@ -26,19 +32,15 @@ class DirCache(MutableMapping):
 
     def __init__(
         self,
-        use_listings_cache=True,
```

Comment:
A cache instance should probably not contain functionality that makes it not work.